aries-site/src/layouts/main/Layout.js

```diff
@@ -53,58 +58,85 @@ export const Layout = ({
       {/* When a backgroundImage is present, the main page content becomes 
       the `last` child. We want this content to drive the layout.
       For details on this prop, see here: https://v2.grommet.io/stack#guidingChild */}
-      <Stack fill guidingChild={backgroundImage && 'last'}>
-        {backgroundImage && (
-          <PageBackground backgroundImage={backgroundImage} />
-        )}
-        <Page>
-          {/* I think Head is redundant at this point, 
+      <ResponsiveContext.Consumer>
+        {size => (
+          <Stack fill guidingChild={backgroundImage && 'last'}>
+            {backgroundImage && (
+              <PageBackground backgroundImage={backgroundImage} />
+            )}
+            <Page>
+              {/* I think Head is redundant at this point, 
               but left it as is for now */}
-          <Head title={title} />
-          <Meta
-            title={title}
-            render={render}
-            description={seoDescription}
-            canonicalUrl={`https://design-system.hpe.design${router.route}`}
-          />
-          <>
-            <SkipLinks id="skip-links">
-              <SkipLink id="main" label="Main Content" />
-            </SkipLinks>
-            <PageContent>
-              <Header fill="horizontal" alignSelf="center" />
-            </PageContent>
-            <MainContentWrapper>
-              <Main overflow="visible">
-                <SkipLinkTarget id="main" />
-                <Box>
-                  {layout !== 'plain' ? (
-                    <>
-                      <ContentSection>
-                        <DocsPageHeader
-                          title={title}
-                          topic={topic}
-                          render={render}
-                        />
-                        {children}
-                      </ContentSection>
-                      {relatedContent.length > 0 && (
-                        <RelatedContent
-                          relatedContent={relatedContent}
-                          title={title}
-                        />
+              <Head title={title} />
+              <Meta
+                title={title}
+                render={render}
+                description={seoDescription}
+                canonicalUrl={`https://design-system.hpe.design${router.route}`}
+              />
+              <>
+                <SkipLinks id="skip-links">
+                  <SkipLink id="main" label="Main Content" />
+                </SkipLinks>
+                <PageContent>
+                  <Header fill="horizontal" alignSelf="center" />
+                </PageContent>
+                <MainContentWrapper>
+                  <Main overflow="visible">
+                    <SkipLinkTarget id="main" />
+                    <Box direction={layout !== 'plain' ? 'row' : 'column'}>
+                      {layout !== 'plain' ? (
+                        <>
+                          <Box
+                            width={
```

Comment:
Maybe we could abstract all of this to something like:

```
const contentWrapperStyles = !['xsmall', 'small'].includes(size) ? { width: ..., pad: ..., margin: ... } : undefined;
```

then spread the `contentWrapperStyles` here

Reply:
also wondering if we might be able to just do an xlarge pad as opposed to using margin too

Reply:
I removed the ResponsiveContext.Consumer and instead declared a size variable above, so that I could implement it this way.
<img width="538" alt="image" src="https://user-images.githubusercontent.com/23144209/181314130-2632f1ac-6389-4480-b043-a9f15cae91a1.png">
<img width="462" alt="image" src="https://user-images.githubusercontent.com/23144209/181315042-be647cb6-cdb2-402e-8067-a56c35044ce3.png">